src/components/form.RadioGroup/form.RadioGroup.tsx

```diff
@@ -0,0 +1,44 @@
+import { Form } from "../form";
+import { useForm } from 'react-hook-form';
+
+type FormData = {
+    diabetesType: string;
+};
+
+
+type DiabetesFormProps = {
+    title: string;
+    description: string;
+    options: string[];
+}
+
+const DiabetesForm = ({title, description, options}: DiabetesFormProps) => {
+    const {register, handleSubmit, formState: { errors }} = useForm<FormData>();
+
+    const onSubmit = (data: FormData) => {
+        if (!errors.diabetesType) {
+        alert(`You selected ${data.diabetesType}`)
+        } else {
+            alert( 'Please select a diabetes type');
+        }
+    };
+
+    return (
+        <Form onSubmit={handleSubmit(onSubmit)}>
+            <p>{title}</p>
```

Comment:
This is great because when exported into another component, we can make the title anything we want in respect to the component we need it for!  😁